be/src/exec/olap_scanner.cpp

```diff
@@ -83,7 +83,7 @@ Status OlapScanner::prepare(
             LOG(WARNING) << ss.str();
             return Status::InternalError(ss.str());
         }
-        _tablet_schema = _tablet->tablet_schema();
```

Comment:
better not copy tablet_schema, because TableSchema does not define copy constructor.  There is a filed  TabletColumn* _parent = nullptr; in tablet column, need change it to shared ptr?

Reply:
Maybe is better to use serizalie to pb and init_from_pb to build a new tablet schema.